packages/language-server/phpUtils/printCraftTwigEnvironment

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+
+namespace Twiggy;
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'getTwigMetadata.php';
+
+[, $WORKSPACE_DIR] = $argv;
+
+$VENDOR_PATH = $WORKSPACE_DIR . DIRECTORY_SEPARATOR . 'vendor';
+
+require_once $WORKSPACE_DIR . '/bootstrap.php';
+
+// TODO: This doesn't disable the error handler.
+const YII_ENABLE_ERROR_HANDLER = false;
+
+/** @var craft\web\Application $app */
+$app = require $VENDOR_PATH . '/craftcms/cms/bootstrap/web.php';
+
+$view = $app->getView();
+$twig = $view->getTwig();
+
+$twigMetadata = \Twiggy\getTwigMetadata($twig, 'craft');
+array_push($twigMetadata['loader_paths'], ...$view->getSiteTemplateRoots());
```

Comment:
`$view->getSiteTemplateRoots()` returns an empty array, and because of that I don't get any completion/definition for paths.

Should we use [getTemplatePaths](https://docs.craftcms.com/api/v4/craft-web-view.html#method-gettemplatespath)?